frontend/src/components/ShootWorkers/ContainerRuntime.vue

```diff
@@ -70,50 +64,63 @@ export default {
     machineImageCri: {
       type: Array,
       default: () => []
+    },
+    kubernetesVersion: {
+      type: String,
+      required: true
     }
   },
   data () {
     return {
       validationErrors,
-      valid: undefined,
-      containerRuntime: undefined,
-      ociRuntime: undefined
+      valid: undefined
     }
   },
   validations,
   computed: {
     containerRuntimeItems () {
-      const containerRuntimes = map(this.machineImageCri, 'name')
-      return uniq([...containerRuntimes, DEFAULT_CONTAINER_RUNTIME])
+      return map(this.machineImageCri, 'name')
     },
     ociRuntimeItems () {
       const containerRuntime = find(this.machineImageCri, ['name', this.containerRuntime])
-      const ociRuntimes = get(containerRuntime, 'containerRuntimes', [])
-      return map(ociRuntimes, 'type')
+      const ociRuntimess = get(containerRuntime, 'containerRuntimes', [])
```

Comment:
I  would also rename `ociRuntimeItems` to `criContainerRuntimeTypes`. This helps me to map the property to the corresponding part in the cloudProfile.

Reply:
ok, done